stostools/Directory.Build.props

```diff
@@ -0,0 +1,132 @@
+		<SToSTestProjects Include="$(RepoSToSSrcDir)\BuildTasks\Tests\*.csproj;$(RepoSToSSrcDir)\Common\Tests\*.csproj" />
+		<SToSProjects Include="$(RepoSToSBuildTaskSrcDir)\*.csproj;$(RepoSToSCommonSrcDir)\*.csproj" />		
+		<SToSNugPkgProjects Include="$(SToSNugPkgSrcDir)\*.csproj" />
+		
+		<NugetPkgProjects Include="@(SToSNugPkgProjects)" />		
+		<ProjectsToBuild Include="@(SToSProjects);@(SToSTestProjects)" />
+		<ProjectsToPublish Include="@(SToSProjects)" />
+	</ItemGroup>
+
+	<!--
+		Detect CI env using available env variables
+		https://<Jenkins server url>/systemInfo
+		https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+	-->
+	<PropertyGroup>
+		<IsJenkinsCI Condition=" '$(JENKINS_HOME)' != '' ">true</IsJenkinsCI>
+		<IsTravisCI Condition=" '$(Travis)' != '' ">true</IsTravisCI>
+		<IsCIEnvironment Condition=" '$(IsJenkinsCI)' == 'true' OR '$(IsTravisCI)' == 'true' ">true</IsCIEnvironment>
+		<IsCIEnvironment Condition=" '$(IsCIEnvironment)' == '' ">false</IsCIEnvironment>
+	</PropertyGroup>
+
+	<!--
+	===========================================================================================
+	Developer Settings
+		On non-windows machine, user can setup an environment variable that will point to their developer settings props file.
+		The name of the file should be named as  currentUser.props (case sensitive)
+		On windows machine, if the environment variable is not set, we will attempt to use the windows current user env. variable
+		
+		Env. variables
+		LocalDevUserBootstrapDirPath == will point to the directory from where currentuser.props should be imported
+		DevMode == flags if you want to be in dev mode, which is basically avoids restoring nuget package and directly loads the build environment
+
+		Default file that will be loaded == currentUser.props (if available)
+	===========================================================================================
+	-->
+	<PropertyGroup Condition=" '$(DevMode)' == 'true' ">
+		<DefaultCurrentUserPropsFileName>currentUser.props</DefaultCurrentUserPropsFileName>		
+		<DeveloperCurrentUserPropFilePath>$(LocalDevUserBootstrapDirPath)\$(DefaultCurrentUserPropsFileName)</DeveloperCurrentUserPropFilePath>
+		<DeveloperCurrentUserPropFilePath Condition=" !Exists($(DeveloperCurrentUserPropFilePath))">$(RepoDeveloperDir)\$(UserName)\$(DefaultCurrentUserPropsFileName)</DeveloperCurrentUserPropFilePath>
+	</PropertyGroup>
+
+	<Import Condition="Exists($(DeveloperCurrentUserPropFilePath)) AND '$(IsCIEnvironment)' != 'true' AND '$(DevMode)' == 'true' " Project="$(DeveloperCurrentUserPropFilePath)" />
+	<PropertyGroup Condition=" '$(IsCIEnvironment)' != 'true' AND '$(DevMode)' == 'true' ">
+		<LocalDevUser Condition=" '$(LocalDevUser)' == '' ">$(UserName)</LocalDevUser>
+		<DefaultCurrentuserTargetsFileName>$(LocalDevUser).bootStrap.targets</DefaultCurrentuserTargetsFileName>
```

Comment:
```suggestion
		<DefaultCurrentUserTargetsFileName>$(LocalDevUser).bootStrap.targets</DefaultCurrentUserTargetsFileName>
```

Reply:
@daschult from the github UI, I am not able to understand what is the comment on the diff.
So can you explain what is that you would like to change.

Reply:
Oh, I expected `Currentuser` to have a capital-u (`CurrentUser`).

Reply:
@daschult you are right your suggestion makes sense to have that proper casing. Will fix it if required in my next iteration, as this file is shared with our common tools, I will need to fix throughout and not just this repository.
Will keep the casing as is for now, but thanks for pointing it out.